packages/techdocs-common/src/stages/publish/azureBlobStorage.ts

```diff
@@ -328,20 +328,22 @@ export class AzureBlobStoragePublish implements PublisherBase {
       const fileExtension = platformPath.extname(filePath);
       const responseHeaders = getHeadersForFileExtension(fileExtension);
 
-      try {
-        this.download(this.containerName, filePath).then(fileContent => {
+      this.download(this.containerName, filePath)
+        .then(fileContent => {
           // Inject response headers
           for (const [headerKey, headerValue] of Object.entries(
             responseHeaders,
           )) {
             res.setHeader(headerKey, headerValue);
           }
           res.send(fileContent);
+        })
+        .catch(e => {
+          this.logger.error(
```

Comment:
That this is not standard now is probably a result of the community-driven nature of publisher development, but it would be good to standardize on a log-level for all of these, while we're here.

I think the intention in the past has been to keep these as `warn`s because they are notable, but not a major failure state of the app.

Reply:
Got it. Didn't actually notice that I had mixed the levels. Great catch 🙏